src/resolvers/registries/npm-resolver.js

```diff
@@ -36,17 +42,21 @@ export default class NpmResolver extends RegistryResolver {
     const satisfied = await config.resolveConstraints(Object.keys(body.versions), range);
     if (satisfied) {
       return body.versions[satisfied];
-    } else {
-      const versions = Object.keys(body.versions);
-      throw new MessageError(
-        config.reporter.lang(
-          'couldntFindVersionThatMatchesRange',
-          body.name,
-          range,
-          (versions.length > 20) ? versions.join(os.EOL) : versions.join(', '),
-        ),
-      );
+    } else if (request && request.resolver && request.resolver.activity) {
+      request.resolver.activity.end();
```

Comment:
Yeah, I figured. No worries. Need to abstract all of these into a `inquirer-reporter` maybe. But, for now it's ok I think.